README.md

```diff
@@ -53,18 +53,58 @@ import { CalendarHeatmap } from '@manufac/reactjs-calendar-heatmap';
   color={color}
   overview={overview}
   handler={print}
+  onTooltip={show}
+  onHideTooltip={hide}
 />
 ```
 
 ### Properties
 
-| Property | Usage                                                                | Default | Required |
-| :------- | :------------------------------------------------------------------- | :-----: | :------: |
-| data     | Time series data from max a year back                                |  none   |   yes    |
-| color    | Theme hex color                                                      | #ff4500 |    no    |
-| overview | Initial overview type (choices are: year, month, day)                |  year   |    no    |
-| handler  | Handler function is fired on click of a time entry in daily overview |  none   |    no    |
+| Property      | Usage                                                                          | Default | Required |
+|:--------------|:-------------------------------------------------------------------------------|:-------:|:--------:|
+| data          | Time series data from max a year back                                          |  none   |   yes    |
+| color         | Theme color in hex code, color names or enum values (choices are: 'spectral' ) | #ff4500 |    no    |
+| overview      | Initial overview type (choices are: global, year, month, week, day)            |  year   |    no    |
+| handler       | Handler function is fired on click of a time entry in daily overview           |  none   |    no    |
+| onTooltip     | onTooltip function is fired on "mouseover" over a visual element               |  none   |    no    |
+| onHideTooltip | onHideTooltip function is fired on "mouseout" over a visual element            |  none   |    no    |
 
+### Props interfaces
+
+#### 1. data
+```
+interface CalendarHeatmapDatum {
+  date: string;
+  total: number;
+  details: { name: string; date: string; value: number }[];
+  summary?: { name: string; value: number }[];
+}
```

Comment:
is this kind of formatting for interface okay @maneetgoyal sir ?

Reply:
no, you should have used "```ts" instead.

however, try to add these interfaces in the same table

Reply:
to include these codes into the table cell sir @maneetgoyal we would need to use `html tables`. markdown table won't work with multiline code. https://stackoverflow.com/questions/28508141/code-block-inside-table-row-in-markdown

Reply:
should we use html table ? or go with whatever formatting we have?

Reply:
experiment and then shortlist what works better

Reply:
Sir we can do it with `<table>` html element, as you can see it is looking good.
![Screenshot from 2022-04-29 17-26-18](https://user-images.githubusercontent.com/79472606/165939743-5fe6b698-6c72-4eba-b427-62625bf312ae.png)

Reply:
the code are little bit mis aligned. it is not in the center relative to other table cells

Reply:
use the table